data/wanderer/wanderers middle.txt

```diff
@@ -20,6 +20,394 @@ mission "Wanderers: Pre-Colony"
 			`	As you land on the outskirts of this settlement, one of the workers comes up to you as you leave your ship. "[Greetings, welcome], it is [delightful, pleasing] to see you have come to show your interest in our exploration. Unfortunately, there exists much [hazard, risk] on a world we know so little about. This will be our staging ground to [explore, catalog] this new region, but it will take some time to be assured of the [safety, security] of our outpost. I suggest you take time to [orient, locate] yourself in these systems while we set up before returning to help. Of course you are welcome when we have [determined, established] safety, there will be many ships needed to help heal the many ruined planets here.`
 				decline
 
+
+mission "Wanderers: Spera Anatrusk Outfitter 1"
+	landing
+	name "Establish outfitter on <origin>"
+	description "Speak to officials at <destination> about establishing an outfitter on <origin>."
+	source "Spera Anatrusk"
+	destination "Vara K'chrai"
+
+	to offer
+		has "event: wanderers: the eye opens"
```

Comment:
Seems like this will trigger immediately following the `Wanderers: Pre-Colony` mission if you visit right away, which can look a little jarring. Should that text be rolled into this mission, so that there's not a jarring flash between two boxes of text?